types/lqip-modern/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for lqip-modern 1.1
+// Project: https://github.com/transitive-bullshit/lqip-modern/
+// Definitions by: Yaroslav Kiliba <https://github.com/Dattaya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.2
+
+/// <reference types="node" />
+
+import sharp = require("sharp");
+
+declare namespace lqip {
+    interface LqipResult {
+        content: Buffer;
+        metadata: {
+            originalWidth: number;
+            originalHeight: number;
+            width: number;
+            height: number;
+            type: string;
```

Comment:
```suggestion
            type: OutputFormat;
```
It can be only one of the supported ouput formats, isn't it?
https://github.com/transitive-bullshit/lqip-modern/blob/master/index.js#L30